plugins/Marketplace/vue/src/PluginList/CTAContainer.vue

```diff
@@ -220,6 +220,11 @@ export default defineComponent({
       type: Boolean,
       required: true,
     },
+    shouldShowInstallLink: {
```

Comment:
Do we need the new prop? Can we just use `plugin.hasDownloadLink` directly in the condition?